plutus-playground-client/default.nix

```diff
@@ -42,6 +42,17 @@ let
     $(nix-build --quiet --no-build-output ../default.nix -A plutus-playground.server-invoker)/bin/plutus-playground webserver
   '';
 
+  # For dev usage
```

Comment:
this should live with the other [convenience scripts](https://github.com/input-output-hk/plutus/blob/master/nix/pkgs/default.nix#L67-L83)

Reply:
To be more precise, it should live under `plutus/nix/pkgs/serve-docs/default.nix` and be referenced from the other convenience scripts. At least that seems sensible to me.

Reply:
@j-mueller thanks for the suggestion; I've moved things around, hopefully it's nicer now :)